packages/react-storage/src/components/StorageBrowser/__tests__/createStorageBrowser.spec.tsx

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import { render, screen, waitFor } from '@testing-library/react';
 
 import * as ProvidersModule from '../providers';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import * as UserAgentModule from '@aws-amplify/core/internals/utils';
```

Comment:
IMO doesn't make sense to disable linting to spy on "core/internals" here when the utility that is being imported and ran in `createStorageBrowser` is declared in the UI package:
```
import { setUserAgent } from '@aws-amplify/ui';
```